bed_maker.py

```diff
@@ -83,26 +83,31 @@ def main():
         cmd = bigWig_template.format(input=args.input_file, output=target, width=width)
     elif args.input_type == "wig": 
         # get path to the genome config; from arg or env var if arg not provided
-        refgenie_cfg_path = select_genome_config(filename=args.rfg_config, check_exist=True, strict_env=True)
+        refgenie_cfg_path = select_genome_config(filename=args.rfg_config, check_exist=False)
         if not refgenie_cfg_path:
             raise OSError("Could not determine path to a refgenie genome configuration file. "
-                          "Use --rfg-config arguemnt or set '{}' environment variable to provide it".
+                          "Use --rfg-config argument or set '{}' environment variable to provide it".
                           format(CFG_ENV_VARS))
-        if not os.path.exists(refgenie_cfg_path):
+        if isinstance(refgenie_cfg_path, str) and not os.path.exists(refgenie_cfg_path):
             # file path not found, initialize a new config file
-            rgc = RGC()
-            rgc.initialize_config_file(filepath=args.rfg_config)
+            print("File '{}' does not exist. Initializing refgenie genome configuration file.".format(refgenie_cfg_path))
+            rgc = RGC(entries={CFG_FOLDER_KEY: os.path.dirname(refgenie_cfg_path)})
+            rgc.initialize_config_file(filepath=refgenie_cfg_path)
```

Comment:
This is the only part I don't understand. I thought you could only create a `refgenconf` object out of a `genome_configuration.yaml` file. Are you first creating the `refgenconf` object as the parent directory and then storing the `genome_configuration.yaml`  in it?

Reply:
`RefGenConf` object can be created out of `entries` (dict), `filepath` (path to the file which is read in) or `yamldata` (yaml-formatted string).  Here, since the file does not exist, I create it with a dict and I get a `RefGenConf` with just the `genome_folder` key populated.

Then I can save the contents of the object wherever I want and it is linked to this file from now on.

Reply:
it's interesting that the constructor doesn't show up in the docs:

http://refgenie.databio.org/en/latest/autodoc_build/refgenconf/

Reply:
I'll look into `lucidoc`. Maybe that's expected and we just need to tweak it

Reply:
it's fixed in `mkdocs-databio` 0.3.2 and `lucidoc` 0.4.2

Reply:
Just tested the new way to initialize the config file. Looks like everything is working fine so I'm going to merge the changes.